lib/ansible/plugins/filter/strftime.yml

```diff
@@ -45,9 +45,9 @@ EXAMPLES: |
   vars:
     date1: '2022-11-15T03:23:13.686956868Z'
     date2: '2021-12-15T16:06:24.400087Z'
-    date_short: '{{ date1|regex_replace("([^.]+)(\.\d{6})(\d*)(.+)", "\1\2\4") }}' #shorten microseconds
+    date1_short: '{{ date1|regex_replace("([^.]+)(\.\d{6})(\d*)(.+)", "\1\2\4") }}' #shorten microseconds
```

Comment:
Can you please move this to https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/filter/to_datetime.yml?